### PR TITLE
Advances #56

### PR DESCRIPTION
**Pull request by cwulfman**

### CODE REVIEW

File: spec/cicognara/tei_indexer_spec.rb [hunk 1]
```diff
@@ -37,6 +37,10 @@
       expect(subject.items.first.n).to eq('1')
     end
 
+    it 'has a title property' do
+      expect(subject.items.first.title).not_to be_nil
+    end
+
```
Comment (jpstroop) [inline spec/cicognara/tei_indexer_spec.rb]: As with an earlier PR that went by, I think you should test that the values are what you expect, e.g.:
```ruby
expected = "Libellus de utilitate et harmonia artium, Antuerpiæ, apud Simonem Cocum"
expect(subject.items.first.title).to eq expected
```

**Comment by cwulfman** [comment]
Sorry to have made a mess here -- I pulled in some conflicts as I was cleaning up preparatory to the PR.

**Comment by jpstroop** [comment]
@cwulfman Should this still be merged? I can help you try to clean it up if you'll be in the office today.

**Comment by cwulfman** [comment]
It's up to you if you want to merge this PR or not.  Do you see any problems with it?

**Comment by jpstroop** [comment]
I'd just like to see the git history cleaned up--one commit message that says what it does would be ideal.

**Comment by jpstroop** [comment]
Closes #56 (should not have been closed before)

**Pull request merged**